pkg/driver_cryptocell/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME=driver_cryptocell
+PKG_URL=https://github.com/Einhornhool/RIOT-cryptocell-driver.git
+PKG_VERSION=24ff1f69ce77bd69ae1d1011daa1a2d238793f23
+
+PKG_LICENSE=ARM Object Code and Header Files License
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+ifneq ($(BOARD),nrf52840dk)
+  $(error This package can only be used with the nRF52840dk)
+endif
```

Comment:
Better use the CPU features to limit the build. As is this won't pass the CI because Murdock will try to build (and fail) this package on other board.
I see that the build limitation logic is implemented in Kconfig (`depends on CPU_MODEL_NRF52840XXAA`) but on the (old) Makefile side (missing `FEATURES_REQUIRED += <something>`, that something could be cryptocell maybe ?).

Reply:
You're right, I forgot to change it in the makefile